pkg/core/klinedriver/klinebuilder.go

```diff
@@ -0,0 +1,295 @@
+package klinedriver
+
+import (
+	"time"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+	log "github.com/sirupsen/logrus"
+)
+
+type Bucket struct {
```

Comment:
You can move Bucket to bucket.go to keep each file concise and focused.

Reply:
will do.

Reply:
done.